src/NzbDrone.Core/Indexers/Definitions/BroadcastheNet/BroadcastheNetRequestGenerator.cs

```diff
@@ -60,15 +60,20 @@ public IndexerPageableRequestChain GetSearchRequests(TvSearchCriteria searchCrit
 
             var btnOffset = searchCriteria.Offset.GetValueOrDefault(0);
 
+            // ID searches act as AND searches, so only one should be used at a time
             if (searchCriteria.TvdbId > 0)
             {
                 parameters.Tvdb = $"{searchCriteria.TvdbId}";
             }
-
-            if (searchCriteria.RId > 0)
+            else if (searchCriteria.RId > 0)
```

Comment:
since sonarr will send both IDs; should we do a search for both IDs

this would mean a Season search would be 4 queries tho which is less than ideal

Reply:
Isn't there some way that Sonarr is checking if aggregate Id search is available and using it if so? I recall something in the nab request code referring to that

Reply:
Yes, but it's a simple check if there are multiple supported params.
https://github.com/Sonarr/Sonarr/blob/83cee4b00e8c64d749d2ff1d8575d31570beecba/src/NzbDrone.Core/Indexers/Newznab/NewznabCapabilitiesProvider.cs#L143

cc @mynameisbogdan